### DISCUSSION TITLE
Bridge all to whatsapp channel. No group

### DISCUSSION THREAD

**Discussion by fabio841** [42wim/matterbridge#2176] [Ideas]
I want ask to admin if possible change code for bridge any message to whatsapp channel (no group)
In this discussion https://github.com/WhiskeySockets/Baileys/issues/549 from other script there is a solution for take a channel ID and use this. It's possible use this ?

**Comment by weskerty** [comment]
Matterbridge uses WhatsMeow
I would think that WhatsMeow is partially compatible with WhatsApp Channels as I've seen many people using WhatsMeow to post to channels.
If you try to add a channel ID channel="ID@newsletter" to matterbridge.toml it will fail with a memory allocation error. Even if you force the installation so that matterbridge uses the latest version of WhatsMeow that won't help, it seems that some changes are required to the matterbridge code.
I hope in the future they add it.
